docs/src/tutorials/standalone/Canopy/changing_canopy_parameterizations.jl

```diff
@@ -1,5 +1,5 @@
 # # Changing Canopy Parameterizations
-# In [Default Canopy](docs/src/tutorials/standalone/Canopy/default_canopy.jl), we ran a simple canopy model simulation
+# In [Default Canopy](@ref "Canopy Model and Parameters"), we ran a simple canopy model simulation
```

Comment:
Im not sure if this is the correct link - I think we want to link to the other tutorial